kong/dao/factory.lua

```diff
@@ -262,6 +262,30 @@ local function default_on_success(identifier, migration_name, db_infos)
       identifier, migration_name)
 end
 
+function _M:are_migrations_uptodate()
+  local migrations_modules = self:migrations_modules()
+  local cur_migrations, err = self:current_migrations()
+  if err then
+    return ret_error_string(self.db.name, nil,
+                            "could not get current migrations: " .. err)
+  end
+
+  for module, migrations in pairs(migrations_modules) do
+    for _, migration in ipairs(migrations) do
+      if not (cur_migrations[module] and 
+          utils.table_contains(cur_migrations[module], migration.name)) then
+
+        local log = require "kong.cmd.utils.log"
+        log.warn(string.format("database is missing migration: (%s) %s", 
+                                module, migration.name))
+        return false, "migrations are not up to date"
```

Comment:
Here the error message should probably contain a better user instruction telling them to start kong with the `--run-migrations` option (on one node only)